lib/fastly.js

```diff
@@ -138,8 +128,24 @@ function handleFastlyResponse(successMessage) {
     const options = {};
     console.log(prettyjson.render(fastlyResponseBody, options));
     if (successMessage) {
-      console.log(chalk.bold.green(successMessage));
+      console.log(chalk.green(successMessage));
     }
     process.stdin.pause();
   };
 }
+
+function fastlyApiKeyPresent(program) {
+  if (!program.apikey) {
+    console.error(chalk.red(fastlyApiKeyError));
+    return false;
+  }
+  return true;
+}
+
+function fastlyServiceIdPresent(program) {
```

Comment:
@ryanhagerty That was one of the ways I tried, but it didn't work. Turns out, I had to move the renamed functions to the top of the file. I guess hoisting doesn't work for arrow functions?